gradle.properties

```diff
@@ -1,5 +1,6 @@
 teamcity-client.version=2.0.47
 components-registry-service-client.version=2.0.32
+component-resolver-core.version=2.0.34
```

Comment:
Do we need it?

Reply:
Yes, this dependency is needed for configuring the components registry. We specifically use `ComposedConfigScript` from this dependency within `Aggregator.groovy`, as well as `BuildSystem` and `RepositoryType` for the component configurations. Without it, the components registry service fails to start properly.